manager/runner.go

```diff
@@ -852,6 +852,9 @@ func (r *Runner) init() error {
 	}
 	log.Printf("[DEBUG] (runner) final config: %s", result)
 
+	//Set VaultDefaultLeaseDuration
+	dep.VaultDefaultLeaseDuration = config.TimeDurationVal(r.config.Vault.DefaultLeaseDuration)
```

Comment:
Not a huge fan of using a module/global variable for this but there is no other good place for now. That is I'm considering this OK for the merge but will probably want to revisit this later... maybe when I port it to hashicat. 

Consider this more an internal note to myself than anything that needs to be changed here.